internal/expand/handler.go

```diff
@@ -76,8 +76,9 @@ type getExpandRequest struct {
 //       404: genericError
 //       500: genericError
 func (h *handler) getExpand(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
-	depth, err := strconv.ParseInt(r.URL.Query().Get("max-depth"), 0, 0)
-	if err != nil {
+	d_ := r.URL.Query().Get("max-depth")
```

Comment:
With go1.17 we can now use `r.URL.Query().Has("max-depth")` https://pkg.go.dev/net/url#Values.Has :wink: 
So I would suggest something along the lines of
```suggestion
	if !r.URL.Query().Has("max-depth") {
		h.d.Writer().WriteError(w, r, herodot.ErrBadRequest.WithError(`required parameter "max-depth" is missing`)
		return
	}
```

Reply:
Sure. I'll add them in a sec.

Reply:
Excuse me for my novice question, but isn't `max-depth` supposed to be optional parameter?

Reply:
Yes, sorry missed that. Thinking about it a second time, it probably makes more sense to make it actually required as I can't come up with a sane default value that makes sense for most use-cases. Currently it is required but that is neither reflected in the API spec, nor is a proper (i.e. helpful) error returned.

Reply:
Ok. I'll make necessary changes and update the patch.